spec/suites/dom/DomUtilSpec.js

```diff
@@ -53,6 +53,31 @@ describe('DomUtil', function () {
 		});
 	});
 
+	describe('getClass', function () {
+		it('should return the element\'s class', function () {
+			el.className = 'bar foo baz';
+			expect(L.DomUtil.getClass(el)).to.eql('bar foo baz');
+		});
+		it('should return empty string if no className', function () {
+			el.id = 'testId';
+			expect(L.DomUtil.getClass(el)).to.eql('');
+		});

```

Comment:
We can omit the test case for en empty class name, this doesn't change anything about the tested implementation compared to the previous test.
```suggestion
```

Reply:
Objection: we expect empty string here. Not `undefined` or so.
I suppose the test make sense.

Reply:
Under which circumstances would `className` be `undefined` here though? I honestly do not see any code paths that would justify this test.

Reply:
> I honestly do not see any code paths that would justify this test.

We write tests not only for current code paths. Imagine tomorrow someone would like to refactor the function.